bundle/Resources/views/themes/admin/site_configuration/content_types.yaml.twig

```diff
@@ -124,3 +124,45 @@
             disable-translation: false
             category: 'content'
             position: 70
+
+# Home page Vide
+-
+    type: content_type
+    mode: create
+    content_type_group: 'Content'
+    identifier: novaezsiteaccessfactory_home_page_vide
```

Comment:
`vide` -> `empty`

Reply:
done for changing identifier by novaezsiteaccessfactory_static_home_page